### DISCUSSION TITLE
5G SA quick start guide?

### DISCUSSION THREAD

**Discussion by TommyA2024** [magma/magma#15424] [General]
Hi, I'm wondering if there is a similar "Quick Start Guide" like the one found at https://magma.github.io/magma/docs/basics/introduction.html for setting up a 5G SA test environment, using AMF instead of MME, etc. (I have access to an isolated radio environment with an Ericsson Baseband 6631 and RRU/Multi MIMO equipment that I would like to connect to a Magma core.)

**Comment by jblakley** [comment]
In general, the procedure for 5G is identical to LTE. There are a few specific things to do after you've got your network up that are outlined here:
https://magma.github.io/magma/docs/lte/integrated_5g_sa
and here;
https://magma.github.io/magma/docs/lte/extended_5g_sa_features

**Reply by TommyA2024** [reply]
Hi, great, thanks! 
BR
/Tommy
